src/sgame/sg_cmds.cpp

```diff
@@ -3166,15 +3167,15 @@ static void Cmd_Tactic_f( gentity_t * ent )
 		if ( Entities::IsAlive( ent ) )
 		{
 			glm::vec3 position = VEC2GLM( ent->s.origin );
-			g_entities[ id ].botMind->userSpecifiedPosition = position;
+			ent->botMind->userSpecifiedPosition = position;
```

Comment:
Here I think this should be `target` instead of `ent`: if I'm not mistaken `ent` represents the user launching the typing command. So they can't be a bot.

Reply:
Nice catch. I tested /tactic and made sure it works now.